addons/website_slides/models/slide_slide.py

```diff
@@ -433,6 +535,20 @@ def write(self, values):
             self.date_published = datetime.datetime.now()
             self._post_publication()
 
+        # avoid fetching external metadata when installing the module (i.e. for demo data)
+        # we also support a context key if you don't want to fetch the metadata when modifying a slide
+        if any(values.get(url_param) for url_param in ['url', 'video_url', 'document_google_url', 'image_google_url']) \
```

Comment:
As the last 3 are related, is it necessary ? Or maybe it calls _write and not write...

Reply:
I tried again putting "url" in the view and printing "values" and I get:

```{'public_views': 0, 'is_published': False, '__last_update': False, 'name': 'sample video several minutes.mp4', 'is_category': False, 'tag_ids': [[6, False, []]], 'active': True, 'channel_id': 1, 'slide_category': 'video', 'slide_type': 'google_drive_video', 'source_type': 'local_file', 'url': False, 'video_url': 'https://drive.google.com/file/d/1qU5nHVNbz_r84P_IS5kDzoCuC1h5ZAZR/view?usp=sharing', 'user_id': 2, 'completion_time': 0.0718, 'is_preview': False, 'slide_resource_downloadable': False, 'description': '<p><br></p>', 'quiz_first_attempt_reward': 10, 'quiz_second_attempt_reward': 7, 'quiz_third_attempt_reward': 5, 'quiz_fourth_attempt_reward': 2, 'sequence': 6}```

As you can see, it passes url to False.
Maybe I'm missing something but it looks like it doesn't work.